tests/gordo_components/builder/test_builder.py

```diff
@@ -512,20 +515,25 @@ def test_provide_saved_model_caching(
     data_config = get_random_data()
     output_dir = os.path.join(tmp_dir, "model")
     registry_dir = os.path.join(tmp_dir, "registry")
-
-    _model, first_metadata = ModelBuilder(
-        name="model-name", model_config=model_config, data_config=data_config
-    ).build(output_dir=output_dir, model_register_dir=registry_dir)
+    machine = Machine(
+        name="model-name", dataset=data_config, model=model_config, project_name="test"
+    )
+    _model, first_metadata = ModelBuilder(machine).build(
```

Comment:
Not your doing, but this is not a private variable, but rather an unused variable. Might change it to "_" as it is not used.

Reply:
I see we do this at multiple places. Change it if you want to, otherwise just ignore me.

Reply:
meh, I think a leading `_` is only "private" if it's an attribute, otherwise it would be pointless to have a private free variable, because no one could actually use it. Anyhow, I can definitely change it, gave it a name only to easily know _what_ it is.

Reply:
I agree, a free private variable makes little sense here anyway, if it was a module it would perhaps be different. It's all convention anyhow, so it really does not matter unless you do import * on a module with "private" functions or something like that. 
So just leave it if you want, does not really matter much.

Reply:
I've fallen in line with your way of thinking, tis now `_` :+1: